ops/pebble.py

```diff
@@ -1687,12 +1916,13 @@ def pull(self,
         # removing opened files, and so we use the tempfile lib's
         # helper class to auto-delete on close/gc for us.
         if os.name != 'posix' or sys.platform == 'cygwin':
-            return tempfile._TemporaryFileWrapper(f, f.name, delete=True)
+            return tempfile._TemporaryFileWrapper(  # type: ignore # noqa
+                f, f.name, delete=True)  # type: ignore # noqa
```

Comment:
`# noqa` seems unnecessary here.

Reply:
I did it to silence pycharm (maybe other IDEs too): it complains that I'm accessing a private attribute in `tempfile`. But, fair enough: EMACS users wouldn't care about that...